src/axom/primal/tests/primal_curved_polygon.cpp

```diff
@@ -74,13 +74,14 @@ primal::CurvedPolygon<CoordType, DIM> createPolygon(
   int iter = 0;
   for(int j = 0; j < num_edges; ++j)
   {
-    std::vector<PointType> subCP;
-    subCP.assign(ControlPoints.begin() + iter,
-                 ControlPoints.begin() + iter + orders[j] + 1);
+    axom::Array<PointType> subCP(orders[j] + 1);
+    for(int i = 0; i < orders[j] + 1; i++) subCP[i] = ControlPoints[i + iter];
```

Comment:
Are we missing `axom::Array::assign()`? 
Would it be worthwhile adding it?

If so, please create an issue for that.

Reply:
We do have a method `axom::Array::set()` which sets a range of values in the `Array`. Maybe it would be worth looking into renaming that to `assign()` to correspond with the `std::vector` interface?

Reply:
See #885